examples/dialin-chatbot/README.md

```diff
@@ -0,0 +1,83 @@
+<div align="center">
+ <img alt="pipecat" width="300px" height="auto" src="image.png">
+</div>
+
+# Dialin example
+
+Example project that demonstrates how to add phone number dialin to your Pipecat bots. We include examples for both Daily (`bot_daily.py`) and Twilio (`bot_twilio.py`), depending on who you want to use as a phone vendor.
+
+- 🔁 Transport: Daily WebRTC
+- 💬 Speech-to-Text (transcription): Daily
```

Comment:
I think it's good to credit Deepgram here. Maybe "Deepgram via Daily transport". wdyt?